example/android/build.gradle

```diff
@@ -27,7 +27,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath("com.android.tools.build:gradle:7.0.4")
+        classpath("com.android.tools.build:gradle:7.2.2")
```

Comment:
I thought react-native was specific about this? But maybe not since this is just a library.

Reply:
Seems to work just fine